src/instruction.rs

```diff
@@ -89,18 +20,18 @@ enum Register {
     D,
     E,
     H,
-    L
+    L,
+    PSW
 }
 
-#[derive(Debug)]
 enum Operand {
```

Comment:
This should be changed to something more descriptive. I have to run errands for the night but will think on it